lib/filterx/expr-variable.c

```diff
@@ -75,21 +50,20 @@ _eval_variable(FilterXExpr *s)
   variable = filterx_scope_lookup_variable(context->scope, self->handle);
   if (variable)
     {
-      FilterXObject *value = filterx_variable_get_value(variable);
+      FilterXObject *value = filterx_scope_get_variable(context->scope, variable);
       if (!value)
         {
           filterx_eval_push_error("Variable is unset", &self->super, self->variable_name);
         }
       return value;
     }
 
-  if (filterx_variable_handle_is_message_tied(self->handle))
```

Comment:
`filterx_variable_handle_is_message_tied()` is no longer used. We should probably remove it to avoid confusion with `FX_VAR_MESSAGE_TIED`

Reply:
good point. fixed.